RHEL/7/input/system/logging.xml

```diff
@@ -50,7 +50,7 @@ logging services, which are essential to system administration.
 </rationale>
 <ident cce="RHEL7-CCE-TBD" />
 <oval id="service_rsyslog_enabled" />
```

Comment:
need to add the XCCDF rule (rsyslog_send_messages_to_logserver) into STIG profile